src/libsync/propagatedownload.cpp

```diff
@@ -778,8 +922,9 @@ void PropagateDownloadFile::downloadFinished()
 
     // In case of conflict, make a backup of the old file
     // Ignore conflicts where both files are binary equal
+    // Ignore items being synced using zsync since a conflict is expected
     bool isConflict = _item->_instruction == CSYNC_INSTRUCTION_CONFLICT
-        && !FileSystem::fileEquals(fn, _tmpFile.fileName());
+        && !FileSystem::fileEquals(fn, _tmpFile.fileName()) && !_item->_isZsync;
```

Comment:
What? How can that be right? A zsync-downloaded can still create a local conflict - why would the method of data transfer make the conflict go away?

Reply:
Not sure why I needed that now, I vaguely remember ._conflict files being created and this being the reason but I don't actually seem to need it now. Removed.